gpt4all-kotlin/src/jvmMain/kotlin/com/xebia/functional/gpt4all/Gpt4AllRuntime.kt

```diff
@@ -38,7 +35,7 @@ suspend inline fun <reified A> AI<A>.getOrThrow(): A = getOrElse { throw it }
  * @see getOrElse for an operator that allow directly handling the [AIError] case.
  */
 suspend inline fun <reified A> AI<A>.toEither(): Either<AIError, A> =
```

Comment:
I don't think this `A` needs to be `reified`? 🤔

Reply:
Probably it doesn't and comes from the original impl, where it was serializing automatically all over.